src/website/src/app/documentation/demos/datagrid/hide-show-columns/hide-show-columns.html

```diff
@@ -70,3 +60,18 @@ <h4>Custom Toggle Strings</h4>
 
 <clr-code-snippet [clrCode]="example"  clrLanguage="html"></clr-code-snippet>
 
+<h4>Custom Toggle Strings</h4>
+
+<clr-alert [clrAlertType]="'danger'" [clrAlertClosable]="false">
+  <div clr-alert-item class="alert-item">
```

Comment:
Is there any behavioral difference between `<div clr-alert-item class="alert-item">` and using `<clr-alert-item>`?

Reply:
IIRC, `<div clr-alert-item...` was the old way. it has been deprecated. As long as it has the `alert-item` class it should still look ok. The CSS is on that classname. But `<clr-alert-item>` is the preferred way to do alerts.

Reply:
Yeah, I copied this from another alert and should update them.